.github/workflows/nix.yaml

```diff
@@ -50,5 +50,6 @@ jobs:
           cd circus-green
           nix flake update charon --override-input charon github:aeneasverif/charon/$CHARON_REV
           nix flake update eurydice --override-input eurydice github:aeneasverif/eurydice/$EURYDICE_REV
+          nix flake update libcrux --override-input libcrux github:cryspen/librux/2ea4f60a8eccaf2cefbc75f6e70264d5019e22ac
```

Comment:
Try this: add `./update-cargo-lock.sh libcrux` before the call to `check.sh`

Reply:
Oh, that failed because `jq` isn't installed... Try `nix develop '.#ci' --command ./update-cargo-lock.sh libcrux` instead?